### ISSUE TITLE
Retrofiting of the RC remote

### ISSUE THREAD

**Issue by joaoafc** [DanielCoelho112/AutoMec-AD#211]
we need a Rc remote to manually drive the car when we are asked. Therefore the new car must be adapted to also be driven manually.

**Comment by Carolf27** [comment]
One idea:
- We can build our own rc radio comand
- There's a raidofrequency module called NRF24L01 
- We can create a wireless conection using 2 of these modules, one conected to the nvidia (i checked and the nvidia has the necessary pins) and the other conected for instance to an arduino nano (this one would be the comand)
- The connections are simple, the modules are cheap (PTrobotics has these modules, we can ask them maybe ;) ), and there are libraries for these modules

Here are some applications of these modules to create RC comands: https://www.youtube.com/watch?v=tzNROquPEHQ and https://howtomechatronics.com/tutorials/arduino/arduino-wireless-communication-nrf24l01-tutorial/?utm_content=cmp-true

This is the NRF24L01 Module:
![image](https://github.com/AutoMecUA/AutoMec-AD/assets/129117353/85434349-97b6-4ace-ab4a-290ca968e553)

It comunicates via radio frequencies. They can be achieved in PTRobotics and AliExpress:
PTRobotics: https://www.ptrobotics.com/nordic-nrf24/5476-1100-meter-long-distance-nrf24l01palna-wireless-module-with-antenna.html?gad_source=1&gclid=EAIaIQobChMIoc_7veL5hQMVnQYGAB0-NQ4BEAQYASABEgKEo_D_BwE
https://www.ptrobotics.com/nordic-nrf24/2537-nrf24l01-module.html?gad_source=1&gclid=EAIaIQobChMIoc_7veL5hQMVnQYGAB0-NQ4BEAQYByABEgKS8fD_BwE

Aliexpress: https://pt.aliexpress.com/item/1005004693699847.html?src=google&src=google&albch=shopping&acnt=631-313-3945&slnk=&plac=&mtctp=&albbt=Google_7_shopping&albagn=888888&isSmbActive=false&isSmbAutoCall=false&needSmbHouyi=false&albcp=19373920267&albag=&trgt=&crea=pt1005004693699847&netw=x&device=c&albpg=&albpd=pt1005004693699847&gad_source=1&gclid=EAIaIQobChMIj8Du5eT5hQMVQahoCR2fmwLZEAQYAyABEgILovD_BwE&gclsrc=aw.ds&aff_fcid=031aa22d47044d54b391d1a2d0b25f48-1715025234601-06830-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=031aa22d47044d54b391d1a2d0b25f48-1715025234601-06830-UneMJZVf&terminal_id=2a1d9392ca9a449c917e9c2f48a94a93&afSmartRedirect=y&gatewayAdapt=glo2bra

Arduino Nano:
AliExpress: https://pt.aliexpress.com/item/1005005708473359.html?src=google&src=google&albch=shopping&acnt=631-313-3945&slnk=&plac=&mtctp=&albbt=Google_7_shopping&albagn=888888&isSmbActive=false&isSmbAutoCall=false&needSmbHouyi=false&albcp=19373920267&albag=&trgt=&crea=pt1005005708473359&netw=x&device=c&albpg=&albpd=pt1005005708473359&gad_source=1&gclid=EAIaIQobChMIipGAlOf5hQMVzpVoCR0f1QG2EAQYASABEgL7v_D_BwE&gclsrc=aw.ds&aff_fcid=272ea977c52f4f27a5bf0f6db93f889b-1715025325889-04809-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=272ea977c52f4f27a5bf0f6db93f889b-1715025325889-04809-UneMJZVf&terminal_id=2a1d9392ca9a449c917e9c2f48a94a93&afSmartRedirect=y

Necessary pins from Nvidia:
![image](https://github.com/AutoMecUA/AutoMec-AD/assets/129117353/d1b93a8c-3061-49f1-95bd-6c4c2f47281e)
1 MISO
1 MOSI
1 SCK
1 CS

In short:  The idea is to build an rc comand that comunicates to the car via radio frequency.
